src/arrayMethodEvery.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomEvery() {
   [].__proto__.every2 = function(callback) {
-    // write code here
+    if (this.length === 0) {
+      return true;
+    }
+
+    let counter = 0;
+
+    for (let i = 0; i < this.length; i++) {
+      if (callback(this[i], i, this)) {
+        counter++;
+      }
```

Comment:
Це можна зробити дещо простіше: Якщо всі юніти масиву, повинні збігтися з результатом колбеку, відповідно, якщо хоча б один з них, йому не відповідає -- можна одразу вертати false.

```suggestion
      if (!callback(this[i], i, this)) {
        return false
      }
```